include/mbedtls/config_adjust_legacy_from_psa.h

```diff
@@ -157,8 +159,9 @@
 #define MBEDTLS_PSA_ECC_ACCEL_INCOMPLETE_KEYTYPES
 #endif
 
+/* Special case: we don't support cooked key derivation in drivers yet */
 #if defined(PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_DERIVE) && \
-    !defined(MBEDTLS_PSA_ACCEL_KEY_TYPE_ECC_KEY_PAIR_DERIVE)
+    (1 || !defined(MBEDTLS_PSA_ACCEL_KEY_TYPE_ECC_KEY_PAIR_DERIVE))
```

Comment:
Non-blocker: I'd prefer to either comment out the `&& !defined(MBEDTLS_PSA_ACCEL_KEY_TYPE_ECC_KEY_PAIR_DERIVE)` than put a `1 ||` short-circuit. That would make it clearer what part of the expression is a temporary limitation. Alternatively, do `#undef MBEDTLS_PSA_ACCEL_KEY_TYPE_ECC_KEY_PAIR_DERIVE`, which would let us have partial support for `MBEDTLS_PSA_ACCEL_KEY_TYPE_ECC_KEY_PAIR_DERIVE` (e.g. in test code): it just would never get effectively activated as long as we don't add the support. But I can live with `1 ||`.